### PR TITLE
Task/frontend242 fix image editor bugs

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
[[Mobile] Image loader modal bugs](https://trello.com/c/koAa2qS9/242-mobile-image-loader-modal-bugs)

File: frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import { Point, Area } from 'react-easy-crop/types';
+import styleControls from '../image-editor/styles.module.scss';
+import ReactSlider from 'react-slider';
+import styles from './styles.module.scss';
+import { Modal } from '../common';
+import { useWindowDimensions } from '../../../hooks/hooks';
+import { getChannelBannerEditorSize, getCroppedImg } from '../../../helpers/helpers';
+import { FC } from '../../../common/types/react/fc.type';
+
+type props = {
```
Comment (WanderningMaster) [inline frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx]: type naming should start with capital letter,

File: frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  banner: string;
+  setBanner: { (newBannerBase64: string): void };
```
Comment (WanderningMaster) [inline frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx]: why you didn't pass handlers something like
handler: () => void,

File: frontend/src/helpers/channel-banner-editor-size/get-channel-banner-editor-size.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  padding: number;
+};
+
+const getChannelBannerEditorSize = ({
+  aspectRatio,
+  screenWidth,
+  padding,
+  controlHeight,
+}: GetChannelBannerEditorSize): Size => {
+  let width = screenWidth - padding;
```
Comment (WanderningMaster) [inline frontend/src/helpers/channel-banner-editor-size/get-channel-banner-editor-size.ts]: i guess you better write something like const width = screenWidth - padding > 1251 ? 1251 :  screenWidth - padding;

**Review by WanderningMaster — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx (hunk 1 above)
Comment (ghost) [reply frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx]: done

File: frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx (hunk 2 above)
Comment (serhii-hnidobor) [reply frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx]: done

File: frontend/src/helpers/channel-banner-editor-size/get-channel-banner-editor-size.ts (hunk 3 above)
Comment (ghost) [reply frontend/src/helpers/channel-banner-editor-size/get-channel-banner-editor-size.ts]: done

File: frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx (hunk 1 above)
Comment (ghost) [reply frontend/src/components/common/channel-banner-editor/channel-baner-editor.tsx]: done

**Review by WanderningMaster — APPROVED** [review]

**Pull request merged**